### ISSUE TITLE
Update batch norm reference in tfjs-node

### ISSUE THREAD

**Issue by nkreeger** [tensorflow/tfjs#1697]
Looks like a new OpDef is ready (and current one is deprecated):

```sh
tf.batchNormalization() is going away. Use tf.batchNorm() instead, and note the positional argument change of scale, offset, and varianceEpsilon You can disable deprecation warnings with tf.disableDeprecationWarnings().
tf.batchNormalization() is going away. Use tf.batchNorm() instead, and note the positional argument change of scale, offset, and varianceEpsilon You can disable deprecation warnings with tf.disableDeprecationWarnings().

```

**Comment by theodddore** [comment]
Hi, could I take a look at this as a first issue or @nkreeger is working on it?

**Comment by nkreeger** [comment]
@theodddore go for it! OpDef registry in core TF is here: https://github.com/tensorflow/tensorflow/blob/master/tensorflow/core/ops/compat/ops_history.v1.pbtxt

We match up those OpDefs in the tfjs node backend for each op here:
https://github.com/tensorflow/tfjs-node/blob/master/src/nodejs_kernel_backend.ts#L1334

**Comment by nkreeger** [comment]
Actually - this might need to be a cleanup in tfjs-core:

```
$ git grep batchNormalization
...
src/ops/batchnorm_test.ts:907:describeWithFlags('deprecated batchNormalization', ALL_ENVS, () => {
src/ops/batchnorm_test.ts:917:    const result = tf.batchNormalization(
src/ops/batchnorm_test.ts:941:    const result2 = tf.batchNormalization2d(
src/tensor.ts:863:  batchNormalization(
src/tensor.ts:869:        'tf.batchNormalization() is going away. ' +
```

**Comment by theodddore** [comment]
Hi, thanks for the tips. I'll start working on it!

**Comment by theodddore** [comment]
Hi @nkreeger,

it seems that batchNormalization from backend is used by batchNorm (using the FusedBatchNorm opDef), so it's my understanding that we should just perform the cleanup you mentioned above in tfjs-core. If what I say is correct, should I remove batchNormalization also from src/tensor.ts, or is it needed as legacy code?

**Comment by OVERSTANDING683** [comment]
As I did my research it needs to be updated

**Comment by mdaj06** [comment]
@nkreeger Hey is this still open?

**Comment by rthadur** [comment]
Related PR has been merged here https://github.com/tensorflow/tfjs/pull/3238 , closing this issue, Thank you
